display/d.mon/render_cmd.py

```diff
@@ -57,6 +71,11 @@ def render(cmd, mapfile):
         env["GRASS_RENDER_FILE"] = mapfile
     try:
         grass.run_command(cmd[0], env=env, **cmd[1])
+        # display driver can generate a blank map file unnecessarily for
+        # non-rendering modules; delete it
```

Comment:
It really seems like it would be the responsibility of non-rendering modules to tell display driver that they will be just printing/managing, but not rendering. ...and for the drivers to have that option in the API. This is fine here, though.

Reply:
Yes, that would be ideal. If all display drivers didn't have any code related to non-rendering management in `*_Graph_set()` and `*_Graph_close()`, it would be a matter of not calling them from `libdisplay`.